packages/validator-ajv8/src/validator.ts

```diff
@@ -202,17 +205,64 @@ export default class AJV8Validator<
    * @private
    */
   private transformRJSFValidationErrors(
-    errors: ErrorObject[] = []
+    errors: ErrorObject[] = [],
+    uiSchema?: UiSchema<T, S, F>
   ): RJSFValidationError[] {
     return errors.map((e: ErrorObject) => {
-      const { instancePath, keyword, message, params, schemaPath } = e;
+      const {
+        instancePath,
+        keyword,
+        params,
+        schemaPath,
+        parentSchema,
+        ...rest
+      } = e;
+      let { message } = rest;
       let property = instancePath.replace(/\//g, ".");
-      let stack = `${property} ${message}`.trim();
```

Comment:
Given how many tests in `core` are failing, it may make sense to either keep this and only replace when title is provided in the `schema` or `uiSchema` OR you'll have to update the core tests to remove the leading `.` that is currently there. I'd lean towards the former since the latter IS a breaking change in behavior vs what the `validator-ajv6` is doing. At the end of the day it might be better to get the opinion of the owner of RJSF, @epicfaace

Reply:
I totally agree.  It is great that there is a full test suite to work against.   I have reverted the default stack.